angular/src/app/shared/app-config/app-config.model.ts

```diff
@@ -9,6 +9,7 @@ export interface AppConfigModel {
   defaultRoute: string;
   googleAnalyticsId: string;
   tosUrl: string;
+  ppUrl: string;
```

Comment:
I know that `tosUrl` is not a great example. However, could you rename `ppUrl` to `privacyPolicyUrl`?